.github/workflows/release.yml

```diff
@@ -0,0 +1,77 @@
+name: Release Opensearch Remote Vector Index Builder to OpensearchProject
+
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: 'Release Version'
+        type: string
+        required: true
+      release_latest_tag:
+            description: 'Publish latest tag'
+            required: true
+            type: boolean
+      release_version_tag:
+            description: 'Publish version tag'
+            required: true
+            type: boolean
```

Comment:
Why use workflow dispatch when we should trigger by cutting a tag.
If you do this way there is no prevention of re-releasing multiple on the same commit.
But also cannot trigger the jenkins workflow because it is listening to the release creation event.